src/form-builder/types.ts

```diff
@@ -17,6 +17,15 @@ export type CheckboxProps = FieldProps<string> & {
   value?: boolean;
 };
 
+export type RadioItemProps = {
```

Comment:
Just realized, I think `RadioGroupProps` is missing from the types file, which is why tests are failing. Can you add? Thanks!